docs/tutorial/part06.rst

```diff
@@ -2,5 +2,648 @@
+this tutorial.
+
+In your ``fetch_api/models/entity.py`` add the following as a method of the ``Entity`` class::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'address': self.address,
+                'jurisdiction': self.jurisdiction,
+                'service_provider': self.service_provider,
+                'countries': self.countries,
+                'jurisdiction_description': self.jurisdiction_description,
+                'valid_until': self.valid_until,
+                'ibcRUC': self.ibcRUC,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'incorporation_date': self.incorporation_date,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+If you look at this closely you'll notice this is just all the properties on the ``Entity`` class
+mapped to a dictionary but the relationships. We'll explain later why we don't serialize the
+relationships here and how to do it, you will need to do the same for the other model classes:
+
+Address::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'country_codes': self.country_codes,
+                'valid_until': self.valid_until,
+                'address': self.address,
+                'countries': self.countries,
+                'node_id': self.node_id,
+            },
+        }
+
+Intermediary::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'valid_until': self.valid_until,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'countries': self.countries,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+Officer::
+
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'valid_until': self.valid_until,
+                'countries': self.countries,
+                'node_id': self.node_id,
+            },
+        }
+
+Other::
+
+    @property
+    def serialize(self):
+        return{
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'name': self.name,
+                'countries': self.countries,
+                'valid_until': self.valid_until,
+                'node_id': self.node_id,
+            },
+        }
+
+Call the serialize methods on the helpers
+=========================================
+
+Now instead of your ``fetch_nodes`` function on your ``fetch_api/utils.py`` looking like this::

```

Comment:
At this point: utils.py doesn't exists yet. I'd to created it from here.

Reply:
@cyuli  the utils.py will we created in part 4; nevertheless, Junior was behind so that's why it not on the tutorial D:

Reply:
Cool :+1: